setup.py

```diff
@@ -40,7 +40,8 @@ def get_install_requires():
         'License :: OSI Approved :: BSD License',
         'Operating System :: OS Independent',
         'Framework :: Django',
-        'Programming Language :: Python :: 2.7',
-        'Programming Language :: Python :: 3.4',
+        'Programming Language :: Python :: 3.5',
+        'Programming Language :: Python :: 3.6',
```

Comment:
Hi ! You can simply do `'Programming Language : : Python : :  3'

Reply:
Sure :smile:

Reply:
Good :smiley: